RInAppMessaging/Classes/Extensions/UIApplication+IAM.swift

```diff
@@ -0,0 +1,31 @@
+import UIKit
+
+extension UIApplication {
+
+    func getKeyWindow() -> UIWindow? {
+        var keySceneWindow: UIWindow?
+        if #available(iOS 13.0, *) {
+            keySceneWindow = connectedScenes
+                .filter({ $0.activationState == .foregroundActive })
+                .compactMap({ $0 as? UIWindowScene })
+                .first?.windows
+                .first(where: { $0.isKeyWindow })
+        }
+
+        return keySceneWindow ?? windows.first { $0.isKeyWindow }
+    }
+
+    func getCurrentStatusBarStyle() -> UIStatusBarStyle? {
+        if #available(iOS 13.0, *),
+           let keyScene = connectedScenes
+                .filter({ $0.activationState == .foregroundActive })
+                .compactMap({ $0 as? UIWindowScene })
+                .first {
+            return keyScene.statusBarManager?.statusBarStyle
+
+        } else {
+            // Not supported
+        }
```

Comment:
You can remove this else {}

Reply:
But then the `// Not supported` comment won't get executed :)
Ok I'll remove else and move the comment to some other place